src/com/oakesville/mythling/MainActivity.java

```diff
@@ -64,7 +64,7 @@ public void onClick(DialogInterface dialog, int which) {
             }).show();
         }
 
-        setContentView(getAppSettings().isTv() ? R.layout.firetv_split : R.layout.split);
+        setContentView(getAppSettings().isFireTv() ? R.layout.firetv_split : R.layout.split);
```

Comment:
not sure I need this, but found it interesting isTv was being checked but a instance of firetv_split was used in the tertiary expression.